test/dummy_luascript.lua

```diff
@@ -0,0 +1,11 @@
+var = 13
+
+if var>13 then
+   print("Hi i have puberty")
```

Comment:
I don't think this is appropriate file content.

Reply:
I am really sorry for this, I will change this thing.